# Represents the PWM value of the left and right motors 0...1
float32 pwm_left
float32 pwm_right